internal/oci/oci.go

```diff
@@ -189,6 +190,22 @@ func (r *Runtime) PrivilegedWithoutHostDevices(handler string) (bool, error) {
 	return rh.PrivilegedWithoutHostDevices, nil
 }
 
+// AllowUsernsAnnotation searches through the AllowedAnnotations for
+// the userns annotation, checking whether this runtime allows processing of "io.kubernetes.cri-o.userns-mode"
+func (r *Runtime) AllowUsernsAnnotation(handler string) (bool, error) {
+	rh, err := r.getRuntimeHandler(handler)
+	if err != nil {
+		return false, err
+	}
+	for _, ann := range rh.AllowedAnnotations {
+		if ann == annotations.UsernsModeAnnotation {
+			return true, nil
+		}
+	}
+
+	return false, nil
+}
+
```

Comment:
IDK, but do you expect there'll be many annotations at some point in the future?  If so, would it make sense to have this live in a new file?  Perhaps `internal/oci/annoctions.go`?

Reply:
there will be more in the future, so I think a new file will make sense, but for now I think I'll leave it here